### DISCUSSION TITLE
Multiple Google Sheets for 1 site

### DISCUSSION THREAD

**Discussion by FotorecordPS** [merchantprotocol/theadslab#58] [User Guide Q&A]
Hi, 

I was wondering if it was possible to create more than 1 spread sheet for a site. Say, have the first spreadsheet have all the parameters from the data collected, then have a 2nd one with only certain data parameters that i could share - a filtered sheet for clients to view in real time. 

Is this possible? I am going to copy the sheet and see if that might work, but I wanted to know if anyone else has tried to do this - if it is actually possible.

**Comment by jonathonbyrdziak** [comment]
Yea it's totally possible. just create another integration and it will create a new spreadsheet for you
